### PR TITLE
SHOR-22: Add contact picture to contact summary page for shoreditch theme for civiCRM

### PR DESCRIPTION
**Pull request by swastikpareek**
## Overview
PR contains code to 
* Add user picture to the contact summary screens
* Add 'semistandard global variable' settings to package.json file.

#### Before/After
shor-22 before
shor-22 after

### Technical Details

* While adding a js file with some Global variables defined semistandard plugin was throwing errors saying variables are not defined (which are actually globally defined.)
 To have this fixed for all JS files we have added semistandard settings to package.json which understand the global variables by default while applying the check.

### Other Details 469e396 
* Moved the whole image div container with the modal preview box functionality. Now on clicking the image the popup open the contact image in preview.
shor-22- enhacement - making the image open in popup on click

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,6 +22,12 @@
       "git add"
     ]
   },
+  "semistandard": {
+    "globals": [
+      "CRM",
+      "jQuery"
```
Comment (AkA84) [inline package.json]: why is jQuery needed? Aren't we following this IIFE-based format everywhere
```js
(function ($) {

}(CRM.$));
```
? If not, any reason you can see for it?

File: scss/civicrm/contact/_header.scss [hunk 2]
```diff
@@ -15,13 +15,16 @@
...
-.crm-summary-contactphoto {
-  border-radius: 50%;
+#crm-contact-thumbnail { 
   float: left;
-  height: $contact-avatar;
   margin: 10px $contact-avatar/2 0;
   padding-top: 5px;
-  width: $contact-avatar;
+
+  img {
```
Comment (AkA84) [inline scss/civicrm/contact/_header.scss]: would a stricter `> img` selector work the same?

File: js/contact-summary.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+(function ($) {
+  $(document).ready(function () {
+    moveUserPictureNextToContactTitle();
+  });
+
+  /**
+   * Prepends the container image div on .crm-summary-contactname-block
```
Comment (AkA84) [inline js/contact-summary.js]: You are just restating the code. Tell me what this function does in practical terms, as I might have no idea what it means to move something in `.crm-summary-contactname-block`

Comment (swastikpareek) [reply js/contact-summary.js]: Got it. Let me update it

File: scss/civicrm/contact/_header.scss (hunk 2 above)
Comment (swastikpareek) [reply scss/civicrm/contact/_header.scss]: Nope. It won't work. `img` is not an immediate child of  `.crm-summary-contactphoto`

File: package.json (hunk 1 above)
Comment (swastikpareek) [reply package.json]: In `js/noConflict` file we are using `jQuery`
```
// https://civicrm.org/licensing
(function($) {
  $.fn.button.noConflict();
})(jQuery);
```

**Comment by swastikpareek** [comment]
Closing this PR as this is on the wrong base fork. The changes have been accounted in the new PR - 
https://github.com/compucorp/org.civicrm.shoreditch/pull/120

**Pull request closed without merging**